go/e2e/handle.go

```diff
@@ -0,0 +1,19 @@
+package function
+
+import (
+	"context"
+	"fmt"
+	"function/resp"
+	"net/http"
+	"os"
+)
+
+func Handle(ctx context.Context, res http.ResponseWriter, req *http.Request) {
+	res.Header().Add("Content-Type", "text/plain")
+	res.WriteHeader(200)
+
+	_, err := fmt.Fprintf(res, resp.Get("REMOTE_VALID")+"\n")
```

Comment:
Here I the purpose of the package is to indirectly ensure func create will download properly folders and content on it (although I saw you put together a test for that along with file mode check). Test/ Build would fail in case that go file is not present.